src/components/atoms/button/button.styles.ts

```diff
@@ -0,0 +1,43 @@
+import { Button, ButtonProps as MuiButtonProps } from '@mui/material';
```

Comment:
Jestem wielkim zwolennikiem przechowywania styli w jednym pliku razem z komponentem.

Twój komponent przycisku ma aktualnie hmmm.... 25 linijek -> możemy to tam wrzucić i nie stracimy na czytelności